### DISCUSSION TITLE
Team members visibility for across different teams.

### DISCUSSION THREAD

**Discussion by gsingh-nuco** [mixpanel/docs-legacy#617] [General]
In one organisation, I'm adding user A to team-A and user B to team-B with organisation role as 'Billing Admin'. Now, user B and their email is visible to user A on `users-and-teams/users` page in `Organization Settings` even when they dont share a common team/project. Is there a way to avoid this?

**Comment by enague** [comment]
Great question! I believe this may be a product gap since seeing who is working on what team may be beneficial, but I'd love to hear your use case and why visibility is important to you. I'd recommend raising this as a product gap here: https://mixpanel.com/submit-feedback
or posting in our #product-ideas slack channel in our community here: https://www.mixpanel.com/community-slack.

Appreciate you flagging this!
